src/test/java/lottery/domain/LotteryTicketsTest.java

```diff
@@ -14,15 +14,15 @@
     @DisplayName("생성 테스트")
     void validGenerationTest() {
         final List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
-        final LotteryTickets generatedTickets = LotteryTickets.create(1, FixedLotteryNumberGenerator.create(numbers));
-        final LotteryTickets expectedTickets = LotteryTickets.create(Arrays.asList(LotteryTicket.of(numbers)));
+        final LotteryTickets generatedTickets = LotteryTickets.create(PurchasePrice.from(1000), FixedLotteryNumberGenerator.create(numbers));
+        final LotteryTickets expectedTickets = LotteryTickets.create(PurchasePrice.from(1000), Arrays.asList(LotteryTicket.from(numbers)));
 
         assertThat(generatedTickets).isEqualTo(expectedTickets);
     }
 
     @Test
     @DisplayName("LotteryNumberGenerator null 테스트")
     void nullArgumentTest() {
-        assertThatIllegalArgumentException().isThrownBy(() -> LotteryTickets.create(1, null));
+        assertThatIllegalArgumentException().isThrownBy(() -> LotteryTickets.create(PurchasePrice.from(1000), (LotteryNumberGenerator) null));
```

Comment:
```suggestion
        assertThatIllegalArgumentException()
        .isThrownBy(() -> LotteryTickets.create(PurchasePrice.from(1000), (LotteryNumberGenerator) null));
```
![image](https://user-images.githubusercontent.com/45464819/146625522-01b2eef5-e2a8-4006-a7ae-d6d243235fd4.png)

어떤 IDE를 쓰시는지는 모르지만... 혹시나 intellij를 쓰신다면 위와 같이 오른쪽에 선이 있을거에요ㅎㅎ
요 선을 넘지 않는 선에서 줄바꿈을 해주시는 걸 추천드립니다!